x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/mappings_state.tsx

```diff
@@ -61,8 +61,8 @@ export const MappingsState = React.memo(({ children, onUpdate, defaultValue }: P
     configuration: {
       defaultValue: defaultValue.configuration,
       data: {
-        raw: {},
-        format: () => ({} as Mappings),
+        raw: defaultValue.configuration,
```

Comment:
When I saw the message in slack I immediately thought about those 2 lines... 😊 Awesome you quickly found the fix to the bug!! @cjcenizal